azure-pipelines.yml

```diff
@@ -13,6 +13,8 @@ pr:
   
 variables:
   projects: '**/*.sln'
+  testsProjects: '**/*Tests/*.csproj'
+  buildConfiguration: $(buildConfiguration)
```

Comment:
remove the whole variables sections, but keep the values for the first two, we're gonna add them through Azure interface https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch